src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="index.html" 
+      method="post" 
+      data-qa="big" 
+      class="form big-bar"
+    >
+      <label class="field">
+        <input 
+        type="search" 
+        name="input" 
+        data-qa="keypress" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input input-big"
+        >
+        <img 
+        src="images/Search.svg" 
+        alt="search" 
+        class="search-icon icon-big"
+        >
+      </label>
+    </form>
+    
+    <form 
+      action="index.html" 
+      method="post" 
+      data-qa="small" 
+      class="form small-bar"
```

Comment:
```suggestion
      class="form form--small"
```
make class names consistent